src/app/tools/symmetries.cpp

```diff
@@ -40,6 +40,10 @@ void HorizontalSymmetry::generateStrokes(const Stroke& mainStroke, Strokes& stro
   Stroke stroke2;
   for (const auto& pt : mainStroke) {
     Stroke::Pt pt2 = pt;
+    if (loop->getDynamics().isDynamic()) {
```

Comment:
Please cache this boolean value to avoid calling getDynamics() on each for() step.
```
bool isDynamic = loop->getDynamics().isDynamic();
for (...
```